recipes/google-cloud-datastore/meta.yaml

```
@@ -0,0 +1,54 @@
+{% set name = "google-cloud-datastore" %}
+{% set version = "1.8.0" %}
+{% set sha256 = "1bc0c6a9af953b71f0956faf775f000076c5631ce32a23f29a89986a84e3bf8b" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pip
+  run:
+    - python
+    - google-cloud-core >=1.0.0
+    - google-api-core-grpc >=1.6.0
+
+test:
+  imports:
+    - google
+    - google.cloud
+    - google.cloud.datastore
+
+about:
+  home: https://github.com/GoogleCloudPlatform/google-cloud-python
+  license: Apache 2.0
```

It's a tiny thing but the SPDX naming scheme puts a dash here:
```suggestion
  license: Apache-2.0
```